--- rocksdb-6.29.5/Makefile.orig	2022-08-21 07:58:56.118421883 +0200
+++ rocksdb-6.29.5/Makefile	2022-08-21 08:00:52.517791294 +0200
@@ -227,6 +227,7 @@ AM_SHARE = $(AM_V_CCLD) $(CXX) $(PLATFOR
 # Export some common variables that might have been passed as Make variables
 # instead of environment variables.
 dummy := $(shell (export ROCKSDB_ROOT="$(CURDIR)"; \
+                  export CFLAGS="$(EXTRA_CFLAGS)"; \
                   export CXXFLAGS="$(EXTRA_CXXFLAGS)"; \
                   export LDFLAGS="$(EXTRA_LDFLAGS)"; \
                   export COMPILE_WITH_ASAN="$(COMPILE_WITH_ASAN)"; \
