--- rocksdb-rocksdb-3.10.2/Makefile~	2015-05-02 02:45:34.000000000 +0300
+++ rocksdb-rocksdb-3.10.2/Makefile	2015-07-27 11:30:48.212446388 +0300
@@ -117,22 +117,23 @@
 #-------------------------------------------------
 # make install related stuff
 INSTALL_PATH ?= /usr/local
+INSTALL_LIBDIR ?= $(INSTALL_PATH)/lib
 
 uninstall:
 	rm -rf $(INSTALL_PATH)/include/rocksdb \
-	  $(INSTALL_PATH)/lib/$(LIBRARY) \
-	  $(INSTALL_PATH)/lib/$(SHARED)
+	  $(INSTALL_LIBDIR)/$(LIBRARY) \
+	  $(INSTALL_LIBDIR)/$(SHARED)
 
 install:
-	install -d $(INSTALL_PATH)/lib
+	install -d $(INSTALL_LIBDIR)
 	for header_dir in `find "include/rocksdb" -type d`; do \
 		install -d $(INSTALL_PATH)/$$header_dir; \
 	done
 	for header in `find "include/rocksdb" -type f -name *.h`; do \
 		install -C -m 644 $$header $(INSTALL_PATH)/$$header; \
 	done
-	[ ! -e $(LIBRARY) ] || install -C -m 644 $(LIBRARY) $(INSTALL_PATH)/lib
-	[ ! -e $(SHARED) ] || install -C -m 644 $(SHARED) $(INSTALL_PATH)/lib
+	[ ! -e $(LIBRARY) ] || install -C -m 644 $(LIBRARY) $(INSTALL_LIBDIR)
+	[ ! -e $(SHARED) ] || install -C -m 644 $(SHARED) $(INSTALL_LIBDIR)
 #-------------------------------------------------
 
 WARNING_FLAGS = -W -Wextra -Wall -Wsign-compare -Wshadow \
